### PR TITLE
Add clientside lua function resetBlurLevel

### PR DESCRIPTION
**Pull request by ghost**
This PR adds clientside method named `resetBlurLevel` which is a starting point for https://github.com/multitheftauto/mtasa-blue/issues/756

### CODE REVIEW

**Review by Woovie — APPROVED** [review]

**Comment by Woovie** [comment]
Any chance we could get resetPlayerBlurLevel as well?

**Comment by ghost** [comment]
> Any chance we could get resetPlayerBlurLevel as well?

Do you mean for server-sided scripts ? I did add this method mainly cause of the above mentioned issue regarding blur level setting. Plus: all (?) `reset*` methods are clientside only.

**Comment by Woovie** [comment]
Yes, server-side too. Nothing about the original issue to me indicates it should be a client method only. Correct me if I'm wrong.

**Comment by ghost** [comment]
Well, no, it didnt indicate that it's meant to be clientside only but i thought it since like i said all `reset*` methods are clientside.

Gonna add `resetPlayerBlurLevel` clientside which is an alias to `resetBlurLevel`.

Imo, we dont need `resetPlayerBlurLevel` on server-side since the server is not responsible for visual stuff unless it's required when it comes to sync (such as weather, time, minute duration)

**Comment by LosFaul** [comment]
resetBlurLevel on server side doesn't make much sence since it's only a visual effect

**Comment by qaisjp** [comment]
why is `resetPlayerBlurLevel` added as just an alias?

**Comment by ghost** [comment]
> why is `resetPlayerBlurLevel` added as just an alias?
Made it an alias because the logic behind it is the same. Kinda hate duplicating code.

**Comment by qaisjp** [comment]
Why do we need both?

**Comment by ghost** [comment]
> Why do we need both?

I dont really think that we need it but @Woovie wanted to have it added as you can see above.

File: Client/mods/deathmatch/logic/lua/CLuaManager.cpp [hunk 1]
```diff
@@ -274,7 +274,6 @@ void CLuaManager::LoadCFunctions()
         {"getGaragePosition", CLuaFunctionDefs::GetGaragePosition},
         {"getGarageSize", CLuaFunctionDefs::GetGarageSize},
         {"getGarageBoundingBox", CLuaFunctionDefs::GetGarageBoundingBox},
-        {"getBlurLevel", CLuaFunctionDefs::GetBlurLevel},
```
Comment (qaisjp) [inline Client/mods/deathmatch/logic/lua/CLuaManager.cpp]: nit: gets and sets are supposed to be placed separately

File: Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.World.cpp [hunk 2]
```diff
@@ -549,6 +549,13 @@ int CLuaFunctionDefs::SetBlurLevel(lua_State* luaVM)
     return 1;
 }
 
+int CLuaFunctionDefs::ResetBlurLevel(lua_State* luaVM)
+{
+    g_pGame->SetBlurLevel(static_cast<unsigned char>(36));
```
Comment (qaisjp) [inline Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.World.cpp]:
```suggestion
    g_pGame->SetBlurLevel(36);
```
just 36 should just work here

**Comment by ghost** [comment]
done @qaisjp

**Review by qaisjp — APPROVED** [review]
:shipit:

**Comment by qaisjp** [comment]
sorry, please see https://github.com/multitheftauto/mtasa-blue/issues/756#issuecomment-599557039

**Pull request merged**